lit-element/src/hello-world.js

```diff
@@ -0,0 +1,3 @@
+import { HelloWorld } from './HelloWorld.js';
```

Comment:
This comment is just a thought, but based on the convention used in the demos from the lit-element team, this file should be named` index.js` and the `HelloWorld.js` as `hello-world.js`

Reply:
This was generated by Open-WC. So I don't know what would be better: keeping the by default naming, or `index.js`. Let me know what you decide.